code/game/turfs/simulated/floor/light_floor.dm

```diff
@@ -103,6 +102,19 @@
 	else
 		to_chat(user, "<span class='warning'>[src]'s light bulb appears to have burned out.</span>")
 
+/turf/simulated/floor/light/proc/toggle_light(state)
+	// 1 = ON
+	if(state == 1)
```

Comment:
Just a `on = state` will do here.

Reply:
Ah yeah, some of this is left over from failed ideas.